server/py/services/alerts/crud/alerts.py

```diff
@@ -48,8 +48,8 @@ def store_alert(
     ):
         project = project or mlrun.mlconf.default_project
 
-        existing_alert = framework.utils.singletons.db.get_db().get_alert(
-            session, project, name
+        existing_alert, existing_alert_state = (
+            framework.utils.singletons.db.get_db().get_alert(session, project, name, with_state=True)
```

Comment:
`existing_alert_state` is not used, so we don't need to query it.

Reply:
@TomerShor i forgot to change enrich_alert method to accept it. Now if alert_state exists, we will pass it to enrich_alert to avoid unnecessary call.

Reply:
Actually, it can also be used here - https://github.com/mlrun/mlrun/pull/7048/files#diff-6809ef5c03708fc9e56be5eb145151e71bc24d022ce9c152293a21e244cdea6aR76-R78